### ISSUE TITLE
Album covers are not viewed

### ISSUE THREAD

**Issue by IjonTheTichy** [owncloud/music#1016]
Like in issue #151:  Album covers what are files named "cover.jpg" in the album folders are not viewed. Version is 1.6.0.

Thanks in advance!

**Comment by paulijar** [comment]
Sorry to hear about your problem. The app is supposed to prioritize using any cover art embedded on the audio file. But if there is no such thing, then it should use any image files found from the same folder with the audio file. The image file names are prioritized in the order `cover.*` > `albumart.*` > `album.*` > `front.*` > `folder.*` > any other image name.

I don't know the reason for your problem but have your tried scanning the music collection from the command line using the `occ music:scan` command? At the end of the scanning process, the command tries to update the cover art images used. If that still doesn't help, then you could try to look from owncloud.log or nextcloud.log if there would happen to be any additional clues.

**Comment by IjonTheTichy** [comment]
Thank you you for your suggestions. I use RPI and dont no, how to use the occ command here. But I found, that only some cover.jpg - files are not viewed but many others.

**Comment by paulijar** [comment]
Is RPI short for RaspberryPi? To use `occ`, you need to connect your server via SSH and navigate to the directory where Nextcloud or ownCloud has been installed. On my system, it's a subdirectory under /var/www/html. There, give command like `sudo -u www-data php ./occ music:scan your_user_name`. That example assumes that the linux user assigned to your web server is `www-data`.

**Comment by IjonTheTichy** [comment]
Yes, its an RaspberryPi. After executing the command you suggested, I got : 
Found 0 music files to scan
Searching cover images for albums with no cover art set...
Searching cover images for artists with no cover art set...

**Comment by paulijar** [comment]
And you still don't see the cover.jpg files which should be there? And you can't see anything interesting in owncloud.log or nextcloud.log?

Just to clarify, do you see the placeholder graphics on those albums (the colorful box with one or two white letters) instead of the proper cover image? Or is there an empty slot where there should be the album cover image?

**Comment by IjonTheTichy** [comment]
Nothing at the log-files. The placeholders are there.

**Comment by paulijar** [comment]
Okay. And when you browse to the relevant folders using the Files app, can you see the cover.jpg files and can you open them without problems?

**Comment by IjonTheTichy** [comment]
Yes.

**Comment by paulijar** [comment]
Kamino closed and cloned this issue to [nc-music/music](https://github.com/nc-music/music/issues/70)
